release.sh

```diff
@@ -34,11 +34,12 @@ function assert_git_state_is_clean() {
 }
 
 function get_release_name() {
-  # DISPLAY_VERSION gets used in the git tag description
-  DISPLAY_VERSION='1.0'
-
   # TAG_NAME gets used as the git tag name
-  TAG_NAME="v${DISPLAY_VERSION}"
+  TAG_NAME="$(./vagrant-spk --version)"
```

Comment:
You can do `$(./vagrant-spk --version | awk '{print $2}')`, which I think is a bit more readable than the magical constant 12 below.

On master we still have the shebang for python 2, where it seems to print the version to stderr, so this won't work there since `$(...)` captures stdout. But it seems to do the right thing on python 3 and I don't think we care about the former -- but it's noteworthy since the python 3 change hasn't actually landed yet.

Reply:
That is probably sensible. The advice I read suggested that what I did, not having used any outside programs, would technically be "faster". But awk is not slow, nor is the release script run often, and I would indeed prefer a single-line solution here.

Re: Python 2/3, my opinion is that we will not/should not push another release dependent on Python 2, and end users should not be running the release script, so yeah, I think I do not care? I was wondering if you felt maintaining Python 2 support was worth retaining, but I would hope that for app developers, being told they need Python 3 is not an exceptional request.

Reply:
Fixed by https://github.com/sandstorm-io/vagrant-spk/pull/293/commits/21a5a79c805be94a72eac5e3cb62eabe20ee2d3c